modules/ROOT/pages/dw-binaries.adoc

```diff
@@ -11,11 +11,11 @@ DataWeave script.
 [%header, cols="1,3"]
 |===
 | Name  | Description
-| xref:dw-binaries-functions-frombase64.adoc[fromBase64] | Transforms a Base64 string into a binary value.
```

Comment:
MDA: Does this output get generated. It is the same as what's in latest src but not clear if had to be manually added.